### DISCUSSION TITLE
notification by device?

### DISCUSSION THREAD

**Discussion by mstewart14** [mundschenk-at/ha-valetudo-blueprints#15] [Q&A] [answered]
I'm quite new to Home Assistant and may be missing something fundamental here but when I set the blueprint up in ha there are no devices to send the notification to... so nothing happens when the notifications are triggered.
I have discord integrated and that provides a service but no devices or entities.  Native notification stuff in a new install also provides no device for this blueprint to use.  
Any suggestions or help is appreciated.

**Comment by mundschenk-at** [comment, marked as answer]
I haven't used the discord integration, but `notify` targets are currently not really entities (this is intended to [change in the near future](https://github.com/home-assistant/architecture/discussions/1041), but not yet). The `Notification Device(s)` selector is intended for the [HA Companion App](https://companion.home-assistant.io) (i.e. your phone). It should be possible to create a [`Notification Group`](https://www.home-assistant.io/integrations/group/#notify-groups) including your Discord service, though.

**Comment by mstewart14** [comment]
Thank you... this is exactly the hinting I needed to move this forward.  Much appreciated.  I did not have the companion app setup and didn't know anything about notification groups.  

For those who might be wanting to setup a notification group, you'll have to hack some yaml in the configuration.yaml file.  
I'm using the discord integration and looks something like 
```
# notify groups
notify:
  - name: group_notify_discord
    platform: group
    services:
      - service: my_discord_service_name
